catalog/src/main/java/org/killbill/billing/catalog/plugin/StandaloneCatalogMapper.java

```diff
@@ -225,6 +225,9 @@ final void populateDefaultCaseChange(final CaseChange input, final DefaultCaseCh
     }
 
     private Iterable<Product> toDefaultProducts(final Iterable<Product> input) {
+        if (input == null) {
```

Comment:
This method is called [here](https://github.com/killbill/killbill/blob/master/catalog/src/main/java/org/killbill/billing/catalog/plugin/StandaloneCatalogMapper.java#L109C16-L109C27), and if you follow the code it seems like we would get an NPE... 

Can you return a `Collections.emptyList()` instead?